pkg/gvk/get.go

```diff
@@ -45,3 +44,18 @@ func Set(obj runtime.Object) error {
 	kind.SetGroupVersionKind(gvks[0])
 	return nil
 }
+
+func ConvertObjs(objs []runtime.Object) ([]schema.GroupVersionKind, error) {
```

Comment:
What is this method for

Reply:
https://github.com/daxmc99/rio/blob/f6f1807b1e27b645519201918e96d4b4b5ec40d6/cli/cmd/up/up.go#L153 
Helper func for handling lists of objects, this will likely go away with a change to variadic functions.